src/test/java/net/sf/xsshtmlfilter/HTMLFilterTest.java

```diff
@@ -1,16 +1,15 @@
 package net.sf.xsshtmlfilter;
 
-import net.sf.xsshtmlfilter.HTMLFilter;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
```

Comment:
No, IntelliJ did it for me and I didn't see it before the commit was done. 
The order is the one we agreed upon in FINN (unless I missed something?), and I though I might as well leave it unless you objected. Do you want me to change it back? The first import import net.sf.xsshtmlfilter.HTMLFilter isn't necessary and can be ommitted.